client/blocks/login/login-form.jsx

```diff
@@ -78,20 +85,21 @@ export class LoginForm extends Component {
 	onSubmitForm = ( event ) => {
 		event.preventDefault();
 
-		const { password, rememberMe, usernameOrEmail } = this.state;
+		const { password, usernameOrEmail } = this.state;
+		let { rememberMe } = this.state;
```

Comment:
I think we're just preparing the input for `loginUser` here https://github.com/Automattic/wp-calypso/pull/16447/files#diff-29356c8042a170020e49bae30afff806R98
This `let` doesn't look too harmful to me